### ISSUE TITLE
Show off screen in the iPhoneX`

### ISSUE THREAD

**Issue by iHTCboy** [naver/cafe-sdk-ios#12]
user cafe-sdk-ios-3.1.0:

![img_dfaecc74e424-1](https://user-images.githubusercontent.com/11418363/40169726-361624a2-59f9-11e8-9f3f-ffdf50f7e68f.jpeg)

Why do you show more, I now how to deal with? plase~

**Comment by hyunminkim81** [comment]
Could you apply SDK version 3.2.4?

**Comment by iHTCboy** [comment]
@hyunminkim81  thank you! let's try ~

**Comment by iHTCboy** [comment]
@hyunminkim81 Our technology is still in the dock, with the result,  thanks~

**Comment by iHTCboy** [comment]
![img_dfc636c889b8-1](https://user-images.githubusercontent.com/11418363/40309360-b6e49812-5d3b-11e8-887a-dc2aede65813.jpeg)


@hyunminkim81  Very sad, still show beyond the interface.

**Comment by iHTCboy** [comment]
@hyunminkim81  ok, Wait me to do a demo, you can have a look

**Comment by g8tauk** [comment]
@iHTCboy Can I get the information below?

- device orientation
**(lldb) p [UIApplication.sharedApplication statusBarOrientation]**
(UIInterfaceOrientation) $4 = UIInterfaceOrientationLandscapeLeft

- screen frame
**(lldb) p [UIScreen.mainScreen frame]**
(CGRect) $5 = (origin = (x = 0, y = 0), size = (width = 812, height = 375))

- window frame
**(lldb) p (CGRect)[UIApplication.sharedApplication.keyWindow frame]**
(CGRect) $6 = (origin = (x = 0, y = 0), size = (width = 812, height = 375))

- safeArea insets
**(lldb) p (UIEdgeInsets)[UIApplication.sharedApplication.keyWindow safeAreaInsets]**
(UIEdgeInsets) $7 = (top = 0, left = 44, bottom = 21, right = 44)

**Comment by iHTCboy** [comment]
@hyunminkim81  @g8tauk  

Because of our game is not really a full screen, so it has a black border around：


```
    if (iPhoneX) {
        self.window = [[UIWindow alloc]initWithFrame:CGRectMake(44, 0, 724, 375)]; 
    }
    else {
        self.window = [[UIWindow alloc]initWithFrame:[UIScreen mainScreen].bounds];
    }
```

**Comment by g8tauk** [comment]
@iHTCboy 
Thank you for a demo.
We adjust layout of a full screen size.
The problem is that a full screen size and a window size are different.
We will fix this problem as soon as possible.

**Comment by iHTCboy** [comment]
@g8tauk Gool ! very happy you can accept, looking forward to your new SDK, thank you, hard you!

**Comment by g8tauk** [comment]
@iHTCboy 
We released 3.2.6
Can you check this version?

**Comment by iHTCboy** [comment]
@g8tauk  Thank you very much, hear the good news, we try the new SDK, problem has been completed, thank you! Hard work!
